app/client/src/features/accounts/PasswordResetRequestForm.tsx

```diff
@@ -0,0 +1,107 @@
+import { makeStyles } from '@material-ui/core/styles';
+import { Form } from '@local/components/Form';
+import { useForm, useSnack } from '../core';
+import { graphql } from 'relay-runtime';
+import { useMutation } from 'react-relay';
+import { PasswordResetRequestFormMutation } from '@local/__generated__/PasswordResetRequestFormMutation.graphql';
+import { LoadingButton } from '@local/components/LoadingButton';
+import { FormContent } from '@local/components/FormContent';
+import { TextField } from '@local/components/TextField';
+
+interface Props {
+    onSuccess: () => void;
+    onFailure: () => void;
+}
+
+const initialState = {
+    email: '',
+};
+
+export type TPasswordResetRequestForm = typeof initialState;
+
+const useStyles = makeStyles((theme) => ({
+    btnGroup: {
+        '& > *': {
+            margin: theme.spacing(1, 0),
+        },
+    },
+    divider: {
+        width: '75%',
+        marginLeft: '12.5%',
+    },
+    avatar: {
+        margin: theme.spacing(1),
+        backgroundColor: theme.palette.secondary.main,
+    },
+    form: {
+        width: '100%', // Fix IE 11 issue.
+        marginTop: theme.spacing(4),
+    },
+}));
+
+const PASSWORD_RESET_REQUEST_FORM_MUTATION = graphql`
+    mutation PasswordResetRequestFormMutation($input: ResetPasswordRequestForm!) {
+        resetPasswordRequest(input: $input) {
+            isError
+            message
+            body
+        }
+    }
+`;
+
+export function PasswordResetRequestForm({ onSuccess, onFailure }: Props) {
+    const classes = useStyles();
+    const [form, errors, handleSubmit, handleChange] = useForm(initialState);
+    const [commit, isLoading] = useMutation<PasswordResetRequestFormMutation>(PASSWORD_RESET_REQUEST_FORM_MUTATION);
+    const { displaySnack } = useSnack();
+
+    function handleCommit(submittedForm: TPasswordResetRequestForm) {
+        commit({
+            variables: { input: submittedForm },
+            onCompleted({ resetPasswordRequest }) {
+                if (resetPasswordRequest.isError) {
+                    displaySnack(resetPasswordRequest.message, { variant: 'error' });
+                    if (onFailure) onFailure();
+                } else {
+                    displaySnack('Success! Please check your email for the password reset link.', {
+                        variant: 'success',
+                    });
+                    if (onSuccess) onSuccess();
+                }
+            },
+        });
+    }
+
+    return (
+        <Grid container justify='center'>
+            <Grid container item xs={12} direction='column' alignItems='center'>
+                <Typography component='h1' variant='h6'>
+                    Reset Password Request
+                </Typography>
+            </Grid>
+            <Form className={classes.form} onSubmit={handleSubmit(handleCommit)}>
+                <FormContent>
+                    <TextField
+                        id='password-reset-request-email'
```

Comment:
What is the id used for here? [Also, may need to set the type to email](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email).